dashboards/x_chain.json

```diff
@@ -750,7 +750,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "Prometheus",
-      "description": "Measures how many of each kind of message are received per second on the X-Chain.",
+      "description": "Measures how long each kind of request on the X-Chain takes to handle.\nThe value for chits, for example, is how long, in seconds, it takes to handle a chits message.",
```

Comment:
nit: can we change `handle` to `process`?

Reply:
Yep.